documentation/text-matching-ranking.html

```diff
@@ -207,6 +207,69 @@ <h3 id="prefix-search">Prefix Search</h3>
 
 
 
+<h2 id="match-configuration">Match Configuration</h2>
+<p>
+Indexed and streaming search differ in execution
+and the match-configuration differs.
+Some times it is useful to inspect generated configuration
+to understand or validate the match configuration.
+Run this to find the value of the -i argument below:
```

Comment:
I don't like this section, it would be better to illustrate with an example instead of using low level shell commands to view generated configuration.

Reply:
moved to end as debug section. will add more match examples for indexed search in next PR

Reply:
I don't see the "move to end" change. Have you pushed it?

Reply:
sorry my bad, pushed now. I'll merge now and follow up in next PR - thanks!